scripts/version.js

```diff
@@ -0,0 +1,154 @@
+const shell = require("shelljs");
+const argv = require("yargs").argv;
+
+if (argv.type) {
+  switch (argv.type) {
+    case "make-stable": {
+      if (argv.fromVersion && argv.toVersion) {
+        const silentState = shell.config.silent; // save old silent state
+        shell.config.silent = true;
+        const fromVersion = argv.fromVersion;
+        const toVersion = argv.toVersion;
+
+        // configuration-api
+        process.stdout.write("Moving configuration-api...");
+
+        shell.mkdir(`./content/management/configuration-api/v${fromVersion}`);
+        shell.exec(
+          `git mv ./content/management/configuration-api/index.mdx ./content/management/configuration-api/v${fromVersion}`
+        );
+        shell.exec(
+          `git mv ./content/management/configuration-api/v${toVersion}/index.mdx ./content/management/configuration-api/`
+        );
```

Comment:
Forgot to add `shell.rm("-R", `./content/management/configuration-api/v${toVersion}`);`